src/driver-steelseries.c

```diff
@@ -462,6 +489,7 @@ steelseries_write_dpi(struct ratbag_resolution *resolution)
 		msg.msg.parameters[2] = i;
 		break;
 	case 2:
+	case 4:
```

Comment:
Is this line necessary? If I understood the code correctly in case of device_version 4 the other method _steelseries_write_resolutions_v4_ is called.